tests/test_end_scenario_sizing_annualized.py

```diff
@@ -0,0 +1,96 @@
+from pathlib import Path
+from unittest import TestCase
+
+import numpy as np
+
+from rtctools.util import run_optimization_problem
+
+
+class TestEndScenarioSizingAnnualized(TestCase):
+    def test_end_scenario_sizing_annualized(self):
+        from models.test_case_small_network_optional_assets_annualized.src.run_annualized import (
+            HeatProblemDiscAnnualizedCost,
+            HeatProblemDiscAnnualizedCostModifiedParam,
+            HeatProblemDiscAnnualizedCostModifiedDiscountRate,
+        )
+        from models.test_case_small_network_optional_assets_annualized.src.run_ates import (
+            HeatProblem,
+        )
+        from models.test_case_small_network_optional_assets_annualized.src import (
+            run_annualized,
+        )
+        from rtctools_heat_network.heat_mixin import calculate_annuity_factor
+
+        base_folder = Path(run_annualized.__file__).resolve().parent.parent
+
+        solution_run_ates = run_optimization_problem(HeatProblem, base_folder=base_folder)
```

Comment:
please add small comment above each "run_optimization_problem" what that problem is, it is not super clear from the naming now

Reply:
Done